src/sensor.cpp

```diff
@@ -476,8 +476,13 @@ void log_callback_end( uint32_t fps,
                         }
                         else
                         {
+                            // The calibration format Y12BPP is modified to be 32 bits instead of 24 due to an issue with MIPI driver
+                            // and padding that occurs in transmission.
+                            // For D4xx cameras the original 24 bit support is achieved by comparing actual vs expected size:
+                            // when it is exactly 75% of the MIPI-generated size (24/32bpp), then 24bpp-sized image will be processed
```

Comment:
The calibration format is supposed to be 24 bpp?
With D457 24 bpp are sent, but packed in a buffer that's sized for 32 bpp, so the last 25% is garbage?
This explanation needs to be very clear.
The cast `(byte*)` on line 473 is a bad idea.